src/navigation/actions.js

```diff
@@ -266,7 +267,10 @@ export const gotoStocktakeEditPage = stocktake => dispatch => {
     params: { title: navStrings.stocktake, stocktake, pageObject: stocktake },
   };
 
-  dispatch(navigationActionCreator(navigationParameters));
+  batch(() => {
```

Comment:
a bunch of these (specifically the pages which can be finalised) action creators have turned into 'dispatcher' action creators (I think this is better then hooking into the `navigation` action in the `FinaliseReducer`